ctapipe_io_magic/__init__.py

```diff
@@ -888,6 +964,157 @@ def parse_metadata_info(self):
             )
 
         return metadata
+        
+    def parse_laser_info(self):
+
+        laser_info_array_list_runh = [
+            'MReportLaser.MReport.fUniqueID',
+            'MReportLaser.MReport.fBits',
+            'MTimeLaser.fMjd',
+            #'MTimeLaser.fTime',
+            'MTimeLaser.fTime.fMilliSec',
```

Comment:
following my comment above I think nanosec and fTelescope you do not need to read.